File: tests/fixtures/php/sample-lib/src/Greeter.php
<?php

namespace PkglyTest\SampleLib;

/**
 * A simple greeter class for integration testing.
 */
class Greeter
{
    /**
     * Returns a greeting message.
     *
     * @param string $name The name to greet
     * @return string A greeting string
     */
    public static function greet(string $name): string
    {
        return "Hello, {$name}!";
    }

    /**
     * Returns the library version.
     *
     * @return string Version string
     */
    public static function getVersion(): string
    {
        return '1.0.0';
    }
}

File: tests/fixtures/go/test-module/greeter.go
// Package testmodule provides simple greeting functionality for integration testing.
package testmodule

import "fmt"

// Version is the module version
const Version = "v1.0.0"

// Greet returns a greeting message for the given name.
func Greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

// GetVersion returns the module version.
func GetVersion() string {
	return Version
}
